Library/Homebrew/cmd/style.rb

```diff
@@ -101,7 +101,7 @@ def check_style_impl(files, output_type, options = {})
       args << "--config" << HOMEBREW_LIBRARY_PATH/".rubocop.yml"
       args << HOMEBREW_LIBRARY_PATH
     else
-      args << "--config" << HOMEBREW_LIBRARY/".rubocop.yml"
+      args << "--config" << HOMEBREW_LIBRARY_PATH.parent/".rubocop.yml"
```

Comment:
We are already relying on this in the same way in the `.rubocop.yml` inside `HOMEBREW_LIBRARY_PATH`:

```yaml
inherit_from:
  - ../.rubocop.yml
```